src/i18n/fi/translation.json

```diff
@@ -19,6 +19,7 @@
   "date-placeholder": "pp.kk.vvvv",
   "delete": "Poista",
   "edit": "Muokkaa",
+  "share": "Jaa",
```

Comment:
This also needs aphabetical ordering and some others.

If you are using VS Code, then the following extension might help to sort values in jsons:

https://marketplace.visualstudio.com/items?itemName=richie5um2.vscode-sort-json